reference/private-networking.html.md

```diff
@@ -97,6 +97,101 @@ Finally, You can discover all the apps in the organization by requesting the TXT
 
 Examples of retrieving this information are in the [fly-examples/privatenet](https://github.com/fly-apps/privatenet) repository.
 
+
+### Example: Ping an app via an `.internal` address
+
+A quick to tutorial to show private networks in action. If you've arrived on private networking, we'll assume you've at least installed the CLI and are at least a little familiar with deploying apps on Fly.
+
+For this example we'll use the [hello-fly](https://github.com/fly-apps/hello-fly) repository.
+
+```cmd
+# clone the repository
+git clone https://github.com/fly-apps/hello-fly
+
+cd hello-fly
+
+# create an app on fly
+fly launch
+
+# output omitted...
+
+? Would you like to deploy now? (y/N)
+N
+
+# deploying now is fine, but we'll want to modify
+# the fly.toml and redeploy again
+```
+
+#### Expose an `internal_port` in the fly.toml
```

Comment:
When I run `fly launch` on `hello-fly` my `fly.toml` looks like this:
```
app = "purple-sound-569"
primary_region = "yyz"

[http_service]
  internal_port = 8080
  force_https = true
  auto_stop_machines = true
  auto_start_machines = true
```
This step should be clear on what needs to be changed in the file, or the example should look like their file?

Reply:
i must've been on an old CLI  version, so i've updated my tutorial.